app/main.py

```diff
@@ -53,6 +53,7 @@ def create_tables(engine, psql_environment):
 app.include_router(invitation.router)
 app.include_router(whatsapp.router)
 app.include_router(search.router)
+app.include_router(keyboard_shortcuts.router)
```

Comment:
Please pull & resolve conflicts :)
After that - please keep alphabetic order

Reply:
Thanks, will do